test/smoke/src/areas/terminal/terminal-tabs.test.ts

```diff
@@ -100,14 +100,14 @@ export function setup() {
 
 		it('should join tabs when more than one non-split terminal', async () => {
 			await terminal.runCommand(TerminalCommandId.Show);
-			await terminal.runCommand(TerminalCommandId.CreateNew);
+			await terminal.runCommandWithValue(TerminalCommandIdWithValue.CreateNew);
```

Comment:
Shouldn't this remain runCommand as it doesn't take a value?

Reply:
it does take a value - "editor"

Reply:
I don't see it getting passed in?